src/Dapr.Client/DaprClient.cs

```diff
@@ -228,6 +228,37 @@ public async ValueTask<StateEntry<TValue>> GetStateEntryAsync<TValue>(string sto
             Dictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Saves the provided <paramref name="stateTransactionRequests" /> to the Dapr state
+        /// store.
+        /// </summary>
+        /// <param name="storeName">The name of the state store.</param>
+        /// <param name="stateTransactionRequests">A list of StateTransactionRequests.</param>
+        /// <param name="metadata">An key/value pair that may be consumed by the state store.  This depends on the state store used.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <typeparam name="TValue">The data type of the value to save.</typeparam>
+        /// <returns>A <see cref="ValueTask" /> that will complete when the operation has completed.</returns>
+        public abstract Task ExecuteStateTransactionAsync<TValue>(
```

Comment:
This method will not take TValue as different states can have different types, its better to take state values as a byte[] and users will serialize the types and pass the values as byte[]

Reply:
fixed